pyproject.toml

```diff
@@ -29,7 +29,7 @@ documentation = "https://docs.camel-ai.org"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<3.12"
 numpy = "^1"
-openai = "^1.2.3"
+openai = "^1.14.0"
```

Comment:
`^1.2.3` means `>=1.2.3, <=2`. The bumping up actually narrows the dependency version range. Please revert it.

Reply:
reverted